### PR TITLE
Converting for payment methods

### LINKED ISSUE #720: [Bug Report] Error to convert when converting order to another currency
## Bug report

The following error occurs when making a currency conversion at the point of sale, where when canceling an order that in this case is in USD, you want to change that amount to the currency in VES to verify what that would be. Amount in Bs based on the rate they handle that day and this is not allowed by the point of sale, since it does not show to change to another currency or the rate, as the other version of ADempiere usually showed

#### Steps to reproduce

The steps to follow for this case are:

1. Have an exchange rate loaded in ADempiere
2. Assemble an order from the point of sale, this can be in dollars or bolivars
3. Apply charge, and in the currency box, try to change to the other currency different from that of the order.

#### Screenshot or Gif（截图或动态图）

Cambiar_Moneda_PDV_AD-vue

#### Link to minimal reproduction（最小可在线还原demo）

https://demo-ui.erpya.com/

#### Expected behavior

The expected result is that the point of sale in the ADempiere-Vue, allows at the time of making the charges to change to a different currency than the order so that it is canceled in a different currency than the point of sale based on the rate that is loaded in ADempiere, as detailed in the following image.

Cambiar_Moneda_PDV_AD3 90

### LINKED ISSUE #736: [Bug Report] Error to add charges to the point of sale
## Bug report

The following error occurs at the time of recording the charges at the point of sale, since when modifying the amount or clicking on the "More" option, it does not currently perform or apply any action.

It should be noted that this problem is occurring right now, since he was working

#### Steps to reproduce
The steps to verify this case are as follows:

1. Enter the point of sale either "USD" or "Ves"
2. Create an order, associating the customer, and several products
3. Click on the payment option
4. When the sale opens to make a record of the charges, click on the "More" option

#### Link to minimal reproduction（最小可在线还原demo）

https://demo-ui.erpya.com/

#### Expected behavior
The expected result for this case is that by giving the option the "more" this allows adding the charge to the point of sale, as it had been doing.

### PR DESCRIPTION
**Pull request by elsiosanchez**
## Bug report
converting for payment methods
#### Steps to reproduce

1. Create sales order
2. Add a Product
3. Add payments with different currencies

#### Screenshot or Gif

dolares

#### Link to minimal reproduction

[demo-ui-erpya](https://deploy-preview-749--stoic-lamport-68decf.netlify.app/)

#### Other relevant information
- Your OS:
- Web Browser:
- Node.js version:
#### Additional context

- Verify that the point of sale can charge in different currencies. 
- Create a conversion rate to date

#### Issues:
Fixes: #720 
Fixes: #736

### CODE REVIEW

**Comment by oscarinak** [comment]
This pull does not fully comply with what is required for price conversion at points of sale and the observations are as follows by point of sale:

1. Point of Sale "VES"

a. When registering the charges and changing the time to "USD" this nowhere shows the rate with which it is working and must show it since it previously showed it.
b. When changing the currency, it effectively converts the amount to dollars according to the rate with which it works that day, however, only a part of that coverion will be added, the balance that remains pending is shown again in bolivars and that is fine, but if desired check that remaining balance again in dollars ADempiere-Vue in this case does not perform a conversion of amounts again
c. The charges associated in a mode different from that of the point of sale when processed and consulted in the "Cash" window, they appear under the same mode of the point of sale and not under the currency in which the charge was associated, which was USD
d. In some cases it does not happen all the time when working with the same payment rule and different currencies, it associates the same charge and the amount to be shown is an amount greater than the total order,in this case you can see the associated gif

Error_PDV-Conversion-VES-USD


2. Point of Sale "USD"

a. It does not perform any conversion, not even when associating the charge and changing the currency, the other currency in which it can be converted is displayed, which in this case would be "You see"

Error_ConversiónPDV-usd-ves-pull-749

**Comment by oscarinak** [comment]
The review was performed again and the following observations were found:

1. For the point of sale "You see"
1.2 It is observed that he places the order and associates the charges when changing the currency, as in the case of "USD", this converts the amount in bolivars to dollars according to the rate that is handled on the day, and the rate is reflected in the part from the superior of the point of sale, as well as as the payments are associated, it will decrease its balance if it is necessary to convert, do it again

Verificación_Multimoneda_PDV-VES-USD

Up to this point all the calculation is done well, but it would be good if the rate value has its identification and decimal separator.

2. For the USD point of sale
2.1 Does not convert to Ves

Error_Convertir_PDV-USD-VES

**Comment by oscarinak** [comment]
The verification was carried out in the following service "https://deploy-preview-749--stoic-lamport-68decf.netlify.app/" where it was observed that the multi-currency is working both for the point of sale "USD" and for The "VES" is attached Gif of the tests carried out:

1. Verification Point of Sale "VES"

Verificación_Conversión_PDV_VES-USD

2. Point of sale verification "USD"

Verificacion_Conversión_PDV-USD-VES

File: src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default [
       size: 24,
       handleActionKeyPerformed: true,
       handleActionPerformed: true,
-      validationCode: 'C_Currency.C_Currency_ID = 100',
+      validationCode: 'C_Currency.C_Currency_ID = 100 OR C_Currency.C_Currency_ID = 50001',
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js:44]: To be change on the next release

**Pull request merged**